### PR TITLE
updated dev container schema URL

### PR DESCRIPTION
**Pull request by thevedicdeveloper**

### CODE REVIEW

File: _implementors/json_reference.md [hunk 1]
```diff
@@ -174,7 +174,7 @@ Variables can be referenced in certain string values in `devcontainer.json` in t
 
 ## <a href="#schema" name="schema" class="anchor"> Schema </a>
 
-You can see the VS Code implementation of the dev container schema [here](https://github.com/microsoft/vscode/blob/main/extensions/configuration-editing/schemas/devContainer.schema.src.json).
+You can see the VS Code implementation of the dev container schema [here](https://github.com/devcontainers/spec/blob/main/schemas/devContainer.base.schema.json).
```
Comment (samruddhikhandale) [inline _implementors/json_reference.md]:
```suggestion
You can see the dev container schema [here](https://github.com/devcontainers/spec/blob/main/schemas/devContainer.base.schema.json).
```

**Review by samruddhikhandale — CHANGES_REQUESTED** [review]
Thank you, left a minor comment.

File: _implementors/json_reference.md (hunk 1 above)
Comment (thevedicdeveloper) [reply _implementors/json_reference.md]: image

I am not sure If I can update it or not.

That's why I just updated the URL link only.

Would you like me to update the change that you mentioned?

Comment (samruddhikhandale) [reply _implementors/json_reference.md]: Apologies for not mentioning that before.
Thanks for updating the URL, and updating the wordings in this PR makes sense to me. 

> Would you like me to update the change that you mentioned?

Yes, that would be great. 
Trying to sync in with https://github.com/devcontainers/spec/pull/257#discussion_r1234420004

Comment (thevedicdeveloper) [reply _implementors/json_reference.md]: updated @samruddhikhandale, please review

**Review by samruddhikhandale — APPROVED** [review]
Thanks!

**Pull request merged**